mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -778,7 +815,9 @@ void mlir::populateArithToEmitCPatterns(TypeConverter &typeConverter,
     ItoFCastOpConversion<arith::SIToFPOp>,
     ItoFCastOpConversion<arith::UIToFPOp>,
     FtoICastOpConversion<arith::FPToSIOp>,
-    FtoICastOpConversion<arith::FPToUIOp>
+    FtoICastOpConversion<arith::FPToUIOp>,
+    FpCastOpConversion<arith::ExtFOp>,
+    FpCastOpConversion<arith::TruncFOp>
```

Comment:
Please excuse me for not having ask this earlier, but what's the motivation to convert to `emitc.cast` instead of to an `emitc.call` that calls `truncf` or even adding a new `emitc.trunc` op? @simon-camp WDYT?